activities/codebase-stewardship/community-assets.md

```diff
@@ -0,0 +1,111 @@
+---
+type: resource
+---
+
+# Community assets
+
+This is a checklist of the most important community assets needed to put an existing codebase on its path to incubation, roughly organized by when they're needed.
+
+## Assumptions
+
+We assume that the chronological goals during incubation are to:
+
+* make sure the codebase is deliberately designed and documented to be reusable
+* create everything needed to enable and support a contributing community
+* focus on collecting evidence and marketing to attract more replicators and contributors
+* improve an active codebase based on performance metrics, user research or new feature requests
```

Comment:
@ElenaFdR, do you think these _need_ trailling punctuation?

Reply:
I've not checked that last box, however if you think no change is necessary then I'd say we're good to go.

Reply:
I do not think these need trailing punctuation.